source/plugins/wakatime/metadata.yml

```
@@ -73,4 +73,9 @@ inputs:
     description: WakaTime username
     type: string
     default: current
-    preset: no
\ No newline at end of file
+    preset: no
+  
+  plugin_wakatime_others:
```

```suggestion
  plugin_wakatime_languages_other:
```
I think it may be more explicit to use `language_other` in this case